### ISSUE TITLE
Game crashes when trying to change anti-aliasing setting during a game

### ISSUE THREAD

**Issue by obstriegel** [inexor-game/code#278]
How to reproduce:
- Start a game
- Change anti-aliasing

Outputs "Caught signal 11 (SIGSEGV)", no stack trace.
It does not happen when you change it in the main-menu.

OS: Ubuntu 14.04.3 LTS
NVIDIA Driver Version: 340.96
master branch (current rev 7650f294c252b97eafbcab76460d8aadd0a3385d)

**Comment by Croydon** [comment]
I can't reproduce this on Windows 7.

**Comment by Fohlen** [comment]
@obstriegel thanks for reporting. I've had a similar issue but couldn't reproduce it any longer. Can you further try to debug the problem with gdb as outlined in the wiki?

**Comment by a-teammate** [comment]
I suggest to close this.
Not really because it is not reproducible currently, but the fucked up systems will disappear with bigger refactorings.
plans are to currently to switch to glfw instead of SDL, to switch the rendering system and the configs are already in flex, not in core.

A new issue for changing settings on-the-fly for core could be opened.
Although I do think we will do this anyways when it is possible, plus submitting setting changes to core only once after editing them in the UI, is probably better placed as note in a "settings UI" ticket.
Anyone mind opening such a "settings UI" ticket, if there isn't any?
@MartinMuzatko maybe? :)
